docs/fundamentals-and-concepts/stellar-ecosystem-proposals.mdx

```diff
@@ -13,7 +13,7 @@ SEPs are publicly created, open-source documents that live in the [GitHub reposi
 
 ## Notable SEPs
 
-There are many SEPs, and they cover a wide variety of standards for interoperation. Whatever you're building, you may want to take a look at the complete list to see if there's a standard for your use case.
+There are many SEPs, and they cover a wide variety of standards for inter-operation. Whatever you're building, you may want to take a look at the complete list to see if there's a standard for your use case.
```

Comment:
We typically don't hyphenate interoperation or any of its forms, we can remove this suggestion